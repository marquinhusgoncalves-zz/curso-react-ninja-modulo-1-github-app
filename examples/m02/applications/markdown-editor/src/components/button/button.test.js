import React from 'react'
import Button from './index'
import renderer from 'react-test-renderer'

it('should Button default to match snapshot', () => {
  const tree = renderer.create(
    <Button onClick={() => null}>
      Click me
    </Button>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

it('should Button success to match snapshot', () => {
  const tree = renderer.create(
    <Button onClick={() => null} kind='success'>
      Click me Success
    </Button>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

it('should Button danger to match snapshot', () => {
  const tree = renderer.create(
    <Button onClick={() => null} kind='danger'>
      Click me Danger
    </Button>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
