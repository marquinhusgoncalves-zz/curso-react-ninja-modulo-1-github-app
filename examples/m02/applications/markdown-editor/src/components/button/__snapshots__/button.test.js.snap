// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should Button danger to match snapshot 1`] = `
<button
  className="button -danger"
  onClick={[Function]}
>
  Click me Danger
</button>
`;

exports[`should Button default to match snapshot 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  Click me
</button>
`;

exports[`should Button success to match snapshot 1`] = `
<button
  className="button -success"
  onClick={[Function]}
>
  Click me Success
</button>
`;
